packages/babel-generator/src/generators/types.js

```diff
@@ -1,6 +1,8 @@
 import * as t from "@babel/types";
 import jsesc from "jsesc";
 
+const surrogates = /[\uD800-\uDBFF][\uDC00-\uDFFF]|([\uD800-\uDBFF]|[\uDC00-\uDFFF])/g;
```

Comment:
nit: Since we are targeting to Node.js 10 from Babel 8, it suffices to use `/\p{Surrogate}/gu`.

Reply:
Ugh, I force pushed the wrong commit. This should be gone now, since jsesc does everything needed.